<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QWidget" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="TomeValue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Chapter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Chapter</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ChapterValue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="Folder">
      <property name="text">
       <string>Folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TextFolder"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="URL">
      <property name="text">
       <string>URL (Must contain [Chapter] and [Image])</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TextURL"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="AddOK">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddCancel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Serie">
      <property name="text">
       <string>Series Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TextSeriesName"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Tome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Tome</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
